src/config/index.ts

```diff
@@ -1,10 +1,14 @@
+const ACCESS_EXPIRES_IN = isNaN(Number(process.env.JWT_ACCESS_TOKEN_EXPIRES_IN))
+	? ''
```

Comment:
```suggestion
const ACCESS_EXPIRES_IN: number = isNaN(Number(process.env.JWT_ACCESS_TOKEN_EXPIRES_IN))
	? 3600
```

みたいにしないとダメそう。もともとnumber型が入ってほしい箇所に空文字(string)詰めちゃってると思う